### PR TITLE
{bio}(foss/2015b) FLASH 1.2.11 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software
+  tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to
+  merge pairs of reads when the original DNA fragments are shorter than twice the length of reads.
+  The resulting longer reads can significantly improve genome assemblies. They can also improve
+  transcriptome assembly when FLASH is used to merge RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['http://download.sourceforge.net/%(namelower)spage']
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb]: use `SOURCEFORGE_SOURCE`

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb (hunk 1 above)
Comment (nathanhaigh) [inline easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb]: I can't as the sourceforge project name differs; it's `FLASHpage` not just `FLASH`.

Comment (boegel) [inline easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2015b.eb]: Ah, ok, sorry, I was already wondering what the `page` thing was doing there, I should have realised. :)

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e1b29d62545da0d709fc for a full test report.

**Comment by boegel** [comment]
Going in, thanks @nathanhaigh

**Pull request merged**